controllers/produtosController.js

```diff
@@ -0,0 +1,60 @@
+const { produtoService } = require('../services');
+const success = 200;
```

Comment:
```suggestion
const HTTP_SUCCESS = 200;
```
Dessa forma fica mais descritivo, e segue a convenção de que constantes definidas em "design time" sejam definidas em maiúscula e snake case.

Se formos aplicar essa sugestão, precisamos tomar o cuidado de mudar mais pra baixo no arquivo, onde essa variável é utilizada